boards/esp32s2-devkit/Makefile.features

```diff
@@ -40,3 +40,7 @@ FEATURES_PROVIDED += periph_pwm
 FEATURES_PROVIDED += periph_spi
 
 FEATURES_PROVIDED += arduino
+
+ifneq (,$(filter esp32s2-devkitc-%,$(BOARD_VERSION)))
+  FEATURES_PROVIDED += tinyusb_device
```

Comment:
Now that's a different PR :wink:

Reply:
Oops. I squashed this very small fix directly.